capabilities.js

```diff
@@ -4,6 +4,19 @@ define('capabilities', ['settings'], function(settings) {
         return !!m && m.matches;
     }
 
+    function detectOS() {
+        var macInfo = /^[^\(]+\(Macintosh.+Mac OS X 10[_\.](\d+)[^\)]+\)/
+            .exec(navigator.userAgent);
```

Comment:
instead of looking at `navigator.userAgent`, can you use `navigator.oscpu` (`"Intel Mac OS X 10.10"`) instead?